ios/RPCModule.swift

```diff
@@ -16,174 +16,206 @@ class RPCModule: NSObject {
       return true
   }
   
+  func getDocumentsDirectory() -> String? {
+    let paths = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)
+    if let pathsFirst = paths.first {
+      return pathsFirst
+    } else {
+      return nil
```

Comment:
@zancas AFAIK when you use the optional type: `String?` you need to return or a `String` or `nil`, here the `nil` value is not a Demon...